primedev/thirdparty/silver-bun/module.cpp

```diff
@@ -204,7 +204,7 @@ CMemory CModule::FindString(const char* szString, const ptrdiff_t nOccurrence, b
 
 	for (size_t i = 0ull; i < m_ExecutableCode.m_nSectionSize - 0x5; i++)
 	{
-		byte byte = pTextStart[i];
+		BYTE byte = pTextStart[i];
```

Comment:
Unrelated change. Probably left over by accident?

Reply:
nope.
I tested this PR by emptying the precompiled header and without it the definition of `byte` is missing here and the compilation would fail.
Upstream was changed a lot so and BYTE makes sense here so I believe this is the right solution.

Reply:
Huh, very curious.

Reply:
found it
its an internal windows typedef that i was not previously aware of
`typedef byte BYTE`

reverted